dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -222,5 +180,11 @@
     = text_field_tag 'level[contained_level_names][]', level_name, placeholder: 'name of contained level'
   #plusanswer
     %i.fa.fa-plus-circle
-  :javascript
-    $("#plusanswer").on("click", function () { $("#plusanswer").prev().clone().insertBefore("#plusanswer") });
+
+%script{src: minifiable_asset_path('js/levels/editors/_blockly.js'),
+        data: {pageOptions: {skin_id: @level.skin,
+                             isK1: @level.try(:isK1),
+                             app: @level.game.app,
+                             solution_blocks: @level.respond_to?(:solution_blocks),
+                             uses_droplet: @level.game.uses_droplet?,
+                             input_output_table: @level.respond_to?(:input_output_table)}.to_json}}
```

Comment:
format nit - too much indent. For readability, I prefer:

```ruby
data: {
  pageOptions: {
    skin_id: @level.skin,
    # ...
  }.to_json
}
```

or maybe extract the inner json, like:

```haml
:ruby
  page_options_json = {
    skin_id: @level.skin,
    # ...
  }.to_json
%script{src: minifiable_asset_path('js/levels/editors/_blockly.js'), data: {pageOptions: page_options_json}
```